packages/website/package.json

```diff
@@ -11,25 +11,34 @@
   "dependencies": {
     "@codesandbox/sandpack-react": "0.10.7",
     "@contentful/f36-components": "^4.0.0",
-    "@contentful/f36-pagination": "^4.1.0-beta",
     "@contentful/f36-core": "^4.0.0",
+    "@contentful/f36-docs-utils": "^4.0.0",
+    "@contentful/f36-icon": "^4.0.0",
     "@contentful/f36-icons": "^4.0.4",
+    "@contentful/f36-pagination": "^4.1.0-beta",
+    "@contentful/f36-tokens": "^4.0.0",
+    "@contentful/f36-utils": "^4.0.0",
+    "@emotion/core": "^10.0.17",
```

Comment:
two emotions? 👀

Reply:
This is because `emotion@10` packages were a mess, basically `@emotion/core` had some functions to interact with the jsx, they even renamed it to `@emotion/react` on the v11.

But the `cx` function that we use, comes from `emotion`, that is the framework agnostic package, so we import it from there, but the `Global` we use for the `GlobalStyles`, comes from `@emotion/core`, so we import from both packages.

And to make things even weirder, both have the `css` method, but the `Global` and the `cx` aren't on both 😓 

Other solution would be to replace the `Global` with the `injectGlobal` and we would be only using the `emotion` package. 🤔

Reply:
uhmmmmm maybe we another PR we try to consolidate that